src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -15,14 +15,15 @@
 
 public class ClimbSubsystem extends BaseSubsystem {
 
-    CANSparkMax winch = new CANSparkMax(RobotConstants.CAN.INTAKE_MOTOR, MotorType.kBrushless);
+    CANSparkMax winch;
     SparkSpeedController winchController;
 
-    private final Solenoid attachHookSolenoid = new Solenoid(RobotConstants.CAN.PCM_ID, RobotConstants.PNEUMATICS.ATTACH_SOLENOID);
-    private final Solenoid engageWinchSolenoid = new Solenoid(RobotConstants.CAN.PCM_ID,
-            RobotConstants.PNEUMATICS.ENGAGE_WINCH);
+    private Solenoid attachHookSolenoid;
+    private Solenoid engageWinchSolenoid;
 
     Timer coord = new Timer();
+    
+    private boolean hardwareAvailability = RobotConstants.AVAILABILITY.climber;
```

Comment:
can this be changed after this point? if not, make it an ALL_CAPS constant.
if you are just doing this to save typing, consider using a static import to save the characters